src/main/java/at/uibk/dps/ee/enactables/modules/FunctionsModule.java

```diff
@@ -1,71 +1,18 @@
 package at.uibk.dps.ee.enactables.modules;
 
-import org.opt4j.core.config.annotations.File;
-import org.opt4j.core.config.annotations.Info;
-import org.opt4j.core.config.annotations.Order;
-import org.opt4j.core.config.annotations.Required;
-import org.opt4j.core.start.Constant;
-import at.uibk.dps.ee.enactables.decorators.DecoratorTimingLogFactory;
 import at.uibk.dps.ee.guice.modules.FunctionModule;
-import ch.qos.logback.classic.util.ContextInitializer;
 
 /**
  * The {@link FunctionsModule} defines the dynamic aspects of the configuration
  * of functions.
- * 
+ *
  * @author Fedor Smirnov
  */
 public class FunctionsModule extends FunctionModule {
 
-  @Order(1)
-  @Info("If checked, the execution time of functions will be logged.")
-  public boolean logTime;
-
```

Comment:
Please delete this file completely. Better to create it again when we need it than having empty classes in the project.

Reply:
Apart from that, everything looks good 👍🏻

Reply:
Removed the file and the resulting empty package.